to-do-list/src/components/PostList/PostList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PostItemInc from './PostItemInc';
+import PostItemComp from './PostItemComp';
+import '../UI/styles/Inc.css'
+import {PostListProps} from '../Interfaces/interfaces'
+
+const PostList = ({posts, titleList, remove, isCompleted}: PostListProps) => {
+    const filteredPosts = posts.filter(post => post.completed === isCompleted);
+
+    if(!posts.length){
+        return (
+            <h3 style={{textAlign:'center'}}>There are no tasks</h3>
```

Comment:
If you'll find no posts from the search, you'll see two same messages 'There are no tasks'